app/test_main.py

```diff
@@ -1,3 +1,47 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ]
+)
+def test_get_human_age_valid(
+        cat_age: int, dog_age: int, expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+def test_get_human_age_negative() -> None:
+    with pytest.raises(ValueError):
+        get_human_age(-1, 10)
+    with pytest.raises(ValueError):
+        get_human_age(10, -5)
+
+
+def test_get_human_age_large_values() -> None:
+    assert get_human_age(10**6, 10**6) == [249996, 199997]
```

Comment:
The expected result `[249996, 199997]` in the `test_get_human_age_large_values` test case may not be accurate due to the issue in the `convert_to_human` function logic identified earlier. Once the logic in `convert_to_human` is corrected, please verify if this expected result still holds true.